company_website/models.py

```diff
@@ -36,6 +38,15 @@ class Employees(models.Model):
     def __str__(self) -> str:
         return self.name
 
+    @property
+    def full_name(self) -> str:
+        return f"{self.name} {self.last_name}"
+
+    def get_display_name(self) -> str:
+        if self.display_name != "":
```

Comment:
If you want to check something like this please use object length -> `if len(self.display_name) < 0`